src/index.js

```diff
@@ -1,13 +1,15 @@
 const express = require('express');
 const env = require('dotenv');
 const { healthRouter } = require('./routes/health.route');
+const { fetchQuotesRouter} = require('./routes/quotes.route')
 
 env.config();
 
 const port = process.env.PORT || 3000;
```

Comment:
Good to have `.env.template` pushed which contains only keys of all env variables used.